src/SprykerEco/Zed/Unzer/Business/UnzerFacadeInterface.php

```diff
@@ -403,4 +404,25 @@ public function expandQuoteWithUnzerCredentials(QuoteTransfer $quoteTransfer): Q
      * @return \Generated\Shared\Transfer\UnzerPaymentTransfer|null
      */
     public function findUpdatedUnzerPaymentForOrder(OrderTransfer $orderTransfer): ?UnzerPaymentTransfer;
+
+    /**
+     * Specification:
+     * - Requires `UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer.quote.unzerCredentials.type` transfer property to be set.
+     * - Requires `UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer.quote.store.name` transfer property to be set.
+     * - Requires `UnzerMarketplacePaymentCredentialsResolverCriteriaTransfer.paymentMethodKey` transfer property to be set.
+     * - Gets main marketplace Unzer credentials from persistence.
```

Comment:
Why gets main if you use criteria? Based on criteria method returns different types of credentials

Reply:
Fixed.